app/controllers/api/responses_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  class ResponsesController < ApplicationController
+
+    def show
+      @response = Response.find(params[:id])
+      respond_with @response
+    end
+
+    def create
+      @response = Response.new form_id: response_params[:form_id], data: response_params[:data]
```

Comment:
Why not response_params with params.require(:response).permit(:form, :data) and Response.new(response_params) ?